src/script.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable import/no-cycle */
 // Import the stylesheet
```

Comment:
- [ ] Same here kindly make sure to fix the linter errors instead of disabling them